File: solucoes/2021/nivel jr/fase 2/js/robo.js
var N, C, S 
var A  
var steps = []

scanf("%d%d%d", "N", "C", "S")

var vzs = 0
var est = 1

for(var i = 0; i < C; i++){
    scanf("%d", "A")
    steps.push(A)
}

if (est == S){
    vzs += 1 
}

for(var i = 0; i < C; i++){
    if (steps[i] == 1){
        est += 1
    }
    else if(steps[i] == -1){
        est -= 1
    }
    
    if (est > N){
        est = 1
    }
    else if(est < 1){
        est = N
    }
        
    if(est == S){
        vzs += 1
    }
}

printf("%d", vzs)

File: solucoes/2021/nivel jr/fase 2/py/robo.py
N, C, S = list(map(int, input().split()))
A = list(map(int, input().split()))

vzs = 0
est = 1

if est == S:
    vzs += 1 

for i in range(len(A)):
        if A[i] == 1:
            est += 1
        elif A[i] == -1:
            est -= 1
    
        if est > N:
            est = 1
        elif est < 1:
            est = N
        
        if est == S:
            vzs += 1

print(vzs)